src/super_gradients/training/utils/checkpoint_utils.py

```diff
@@ -44,6 +44,12 @@ def transfer_weights(model: nn.Module, model_state_dict: Mapping[str, Tensor]) -
             pass
 
 
+def maybe_remove_module_prefix(state_dict):
```

Comment:
I think it's quiet clear, but a simple docstring with an example is always better